common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -78,7 +79,7 @@
                             @fragments.inlineSvg("search-36", "icon", List("main-menu__icon", "main-menu__icon--search"))
                         </label>
 
-                        <label for="q"
```

Comment:
This is actually important. This is how we search on Google 😆 
![image](https://cloud.githubusercontent.com/assets/8774970/24958572/ac185a6a-1f87-11e7-9c0a-9eddc5877c25.png)